static/js/stream_popover.js

```diff
@@ -314,6 +314,7 @@ function build_topic_popover(opts) {
         html: true,
         trigger: "manual",
         fixed: true,
+        fix_positions: true,
```

Comment:
I couldn't find documentation for this ancient bootstrap flag, but I did find this comment in `static/third/bootstrap-tooltip/tooltip.js`, which suggests this is likely not correct?

/* Ensure that the popover stays fully onscreen,
   as best as we can.  It might still not look  
   great--in some cases, we should probably just
   center--but this patch makes the popover more
   likely to be usable.  (If the screen is super
   small, obviously we can't fit it completely.)
                                                
   If you use this fix_positions option, you want
   to also use the "no_arrow_popover" template. 
*/                                              

In any case, I think I'd prefer to see us migrate this popover to the `popover_menus.js` system (which doesn't use ancient Bootstrap) over continuing to fiddle with Bootstrap configuration; I think that's likely not super hard.

Reply:
Great then, I will migrate this popover system into `popover_menus.js`